src/components/EngineManageDialog.vue

```diff
@@ -150,31 +155,46 @@
           >
             <div class="q-pl-md q-mt-md flex">
               <img
+                v-if="engineIcons[selectedId]"
```

Comment:
意図としてはこちらのほうがわかりやすいかも
```suggestion
                v-if="selectedId in engineIcons"
```